### PR TITLE
Updated 'what happens next'

### PR DESCRIPTION
**Pull request by jamesrose**
### Context

Superseding old PR https://github.com/DFE-Digital/early-careers-framework/pull/351

### CODE REVIEW

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 1]
```diff
@@ -6,16 +6,10 @@
 
     <h2 class="govuk-heading-m">What happens next</h2>
     <ul class="govuk-list govuk-list--bullet">
-      <li>your school will receive an email with a link to nominate an induction tutor</li>
-      <li>this link will expire in 21 days</li>
+      <li>sent to the email address your school has listed on our register, <%= govuk_link_to "Get information about schools", "https://www.get-information-schools.service.gov.uk/" %></li
```
Comment (blackrat) [inline app/views/nominations/request_nomination_invite/success.html.erb:9]: "www" not required on the url

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 2]
```diff
@@ -6,16 +6,10 @@
...
       <li>
-        we cannot disclose the email address. It will be an administration email, for example, office@, admin@ or head@
+        your school must follow this link to nominate an induction lead or tutor
       </li>
-      <li>
-        if your school does not receive this email, ensure your school email address is correct on our register,
-        <%= govuk_link_to "Get information about schools", "https://www.get-information-schools.service.gov.uk/" %>
```
Comment (blackrat) [inline app/views/nominations/request_nomination_invite/success.html.erb:16]: although it's not actually your change that had it there. :)

File: app/views/nominations/request_nomination_invite/success.html.erb (hunk 1 above)
Comment (ethanmills) [inline app/views/nominations/request_nomination_invite/success.html.erb:9]: Looks like we dropped a `>` on the closing tag here. I suspect that what Cypress is annoyed about

Comment (jamesrose) [reply app/views/nominations/request_nomination_invite/success.html.erb]: @ethanmills Ah, great spot! I must have copied it over from the old PR incorrect.

**Review by ethanmills — APPROVED** [review]

**Pull request merged**